src/components/billingRecord/IFXCalculateBillingMonth.vue

```diff
@@ -139,25 +200,20 @@ export default {
     clearInterval(this.interval)
     next()
   },
-  watch: {
-    selectedDate() {
-      if (this.selectedDate) {
-        this.getUsages()
-      }
-    }
-  },
   mounted() {
     if (this.month && this.year) {
       this.selectedDate = `${this.year}-${this.month}`
+    } else {
+      this.selectedDate = this.$api.storage.getItem(this.selectedDateKey, 'session') || null
     }
+    this.search = this.$api.storage.getItem(this.searchStorageKey, 'session') || ''
```

Comment:
One thing to check; with `NICE`, if I deleted the search string, it wrote `"null"` into the session storage. When the page reloaded, it was searching for the literal string "null" which usually returned nothing.

Reply:
IFXAPI storage takes care of this.